lms/static/sass/course/modules/_calculator.scss

```diff
@@ -27,17 +30,18 @@ div.calc-main {
     width: 16px;
 
     &:hover, &:focus {
-      opacity: 0.8;
+      background-color: $black;
     }
 
     &.closed {
       background-image: url("../images/close-calc-icon.png");
```

Comment:
Yeah, it should be cleaned up, but its out of scope for this PR which focuses on adding/styling the toggle notes UI. Happy to have a story added to the backlog to clean it up if you want to at a later time.